### DISCUSSION TITLE
Is Tenacity still developed?

### DISCUSSION THREAD

**Discussion by ghost** [tenacityteam/tenacity-legacy#738] [Q&A] [answered]
Is this project still being actively developed? If yes, why is it called 'tenacity-legacy'?

**Comment by poyribn** [comment, marked as answer]
Tenacity stopped being developed after the team lost motivation. Then, a person made [a fork named Saucedacity](https://github.com/tenacityteam/saucedacity) which is actively developed by another team, which now includes some of the original Tenacity team.
Now the Saucedacity and Tenacity teams have agreed to turn Saucedacity into Tenacity with a 2-step plan:
1. transfer the Saucedacity repository to the Tenacity organisation (Done)
2. rebrand Saucedacity to Tenacity (in progress)

to preserve the original Tenacity repository, they renamed it into tenacity-legacy.

**Reply by ghost** [reply]
Got it, thanks for the confirmation. 👍

**Reply by n0toose** [reply]
To be clear, I'm just there to give advice and get some of the bureaucratic stuff that the previous team was responsible for and the Saucedacity fork existed beforehand -- but has now built most of our work into this.
